ccloud/beginner-cloud/docker-compose.yml

```diff
@@ -2,7 +2,7 @@
 version: '2'
 services:
   connect-cloud:
-    image: cnfldemos/cp-server-connect-datagen:0.1.7-5.4.x-latest
+    image: cnfldemos/cp-server-connect-datagen:0.1.7-5.4.0
```

Comment:
https://confluent.slack.com/archives/CSTVAFTCN/p1579203914036700

Reply:
we don't need to hold up the PR for this, will take me a bit to build and publish them